src/file_utils.rs

```diff
@@ -104,3 +104,59 @@ pub(crate) fn decode_resource_data_entry(yaml_location: &YamlLocation, value_at_
     }
     .clone())
 }
+
+pub(crate) fn add_annotation(resource: &mut Value, yaml_location: &YamlLocation) -> Result<()> {
```

Comment:
nit: function name feels a bit too general

```suggestion
pub(crate) fn add_recert_edited_annotation(resource: &mut Value, yaml_location: &YamlLocation) -> Result<()> {
```